tests/python/release_checklist/check_static_components_ui.py

```diff
@@ -6,7 +6,7 @@
 
 import rerun as rr
 
-README = """
+README = """\
```

Comment:
what happend on these? I'm not python enough to understand this slash

Reply:
Remove the leading newline of all the readmes. Before the sinle-line summary of the readmes was "…"